packages/docgen/src/documentation.ts

```diff
@@ -123,7 +123,7 @@ export class Documentation {
 
 					case 'Method': {
 						const event = prop?.groups?.find((group) => group.title === 'Events');
-						if ((event?.children as unknown as number[])?.includes(member.id)) {
+						if ((event?.children as unknown as number[])?.includes(member.user.id)) {
```

Comment:
This is wrong.

Reply:
I'm assuming this has to be reverted—I think I accidentally missed the fact that this was not an actual `GuildMember`